docs/data/base/components/trap-focus/ContainedToggleTrappedFocus.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+import Box from '@mui/material/Box';
+import Stack from '@mui/material/Stack';
+import TrapFocus from '@mui/base/TrapFocus';
+
+export default function BasicTrapFocus() {
+  const [open, setOpen] = React.useState(false);
+
+  return (
+    <Box
+      sx={{
+        display: 'flex',
+        alignItems: 'center',
+        flexDirection: 'column',
+      }}
+    >
+      <TrapFocus open={open} disableRestoreFocus disableAutoFocus>
+        <Stack>
+          <Stack alignItems="center">
```

Comment:
It would have been simpler like this, we don't need two Stack.

```suggestion
        <Stack alignItems="center" spacing={2}>
```

Reply:
That makes sense!